### PR TITLE
Fix bug 1482067 - Do not fail when loading caighdean while offline.

### PR DESCRIPTION
**Pull request by adngdb**

### CODE REVIEW

File: pontoon/machinery/views.py [hunk 1]
```diff
@@ -142,6 +146,12 @@ def machine_translation_caighdean(request):
     except Translation.DoesNotExist:
         return JsonResponse({})
 
+    if Translator is None:
+        return http.HttpResponseServerError(
+            json.dumps(dict(error=500, message='Caighdean unavaible offline')),
```
Comment (jotes) [inline pontoon/machinery/views.py]: `unavailable`

File: pontoon/machinery/views.py [hunk 2]
```diff
@@ -148,7 +150,10 @@ def machine_translation_caighdean(request):
 
     if Translator is None:
         return http.HttpResponseServerError(
-            json.dumps(dict(error=500, message='Caighdean unavaible offline')),
+            json.dumps({
+                'error': 500,
+                'message': 'Caighdean unavailable offline'
```
Comment (mathjazz) [inline pontoon/machinery/views.py]: Nit: comma. :P

**Comment by adngdb** [comment]
I realized just before merging that this is exactly the kind of code that requires a comment. Pretty sure 31-year-old me won't understand this without the comment I just added.

**Comment by mathjazz** [comment]
Looks like we forgot to merge this.

**Pull request merged**